app/views/contexts/hub_context.py

```diff
@@ -50,18 +50,21 @@ def get_context(self, survey_complete, enabled_section_ids) -> Mapping:
             submit_button = hub_schema.get("submission", {}).get(
                 "button"
             ) or lazy_gettext("Submit survey")
+            submission_guidance = hub_schema.get("submission", {}).get("guidance", "")
 
         else:
             title = lazy_gettext("Choose another section to complete")
             guidance = custom_text.get("guidance") or lazy_gettext(
                 "You must complete all sections in order to submit this survey"
             )
             submit_button = lazy_gettext("Continue")
+            submission_guidance = ""
```

Comment:
We should use `NoneType` over empty strings for non-existent keys. This also makes it consistent with the current pattern.
`submission_guidance = hub_schema.get("submission", {}).get("guidance")`
`submission_guidance = None`